fe1-web/src/core/network/ingestion/Handler.ts

```diff
@@ -61,6 +65,25 @@ export function handleExtendedRpcRequests(req: ExtendedJsonRpcRequest) {
         req.receivedFrom,
       ),
     );
+  } else if (req.request.method === JsonRpcMethod.GREETING) {
+    const greetingParams = req.request.params as Greeting;
+
+    dispatch(
+      addServer(
+        new Server({ address: greetingParams.address, publicKey: greetingParams.sender }).toState(),
+      ),
+    );
+
+    // connect to all received peer addresses
+    // IMPORTANT: The network manager deduplicates connections to the same address (string)
+    // and the received peer addresses are supposed to be the canonical ones.
+    // Hence it just has to be made sure that the first connection also is to the canonical
```

Comment:
```suggestion
    // Hence we just have to make sure that the first connection is also to the canonical
```
(tiny bit simpler / clearer)

Reply:
👍